app/provider.go

```diff
@@ -32,5 +32,9 @@ func ProvideOauthTokenSource(githubToken GithubToken) oauth2.TokenSource {
 }
 
 func ProvideOauthClient(ts oauth2.TokenSource) *http.Client {
-	return oauth2.NewClient(oauth2.NoContext, ts)
+
+	return oauth2.NewClient(
+		oauth2.NoContext,
```

Comment:
SA1019: oauth2.NoContext is deprecated: Use context.Background() or context.TODO() instead.  (staticcheck)